src/index.ts

```diff
@@ -439,31 +479,32 @@ export default class D2Reader {
   get positions() {
     return this.navigator.positions();
   }
-  goTo = async (locator: Locator) => {
+  goTo = (locator: Locator) => {
     this.navigator.goTo(locator);
   };
-  goToPosition = async (value: number) => {
+  goToPosition = (value: number) => {
     return this.navigator.goToPosition(value);
   };
-  nextResource = async () => {
+  nextResource = () => {
     this.navigator.nextResource();
   };
-  previousResource = async () => {
+  previousResource = () => {
     this.navigator.previousResource();
   };
-  nextPage = async () => {
+  nextPage = () => {
     this.navigator.nextPage();
   };
-  previousPage = async () => {
+  previousPage = () => {
     this.navigator.previousPage();
   };
-  atStart = async () => {
+  atStart = () => {
     return this.navigator.atStart();
   };
-  atEnd = async () => {
+  atEnd = () => {
     return this.navigator.atEnd();
   };
-  snapToElement = async (value: HTMLElement) => {
+  // currently not used or functional
+  snapToElement = (value: HTMLElement) => {
     this.navigator.snapToElement(value);
   };
```

Comment:
Let's just remove it if it's not functional I think

Reply:
well, i will need it soon though. i didn't want to remove it, and then forget about it, that's why the comment. we can comment it out maybe

Reply:
Oh I see I thought it was just not going to be used 👍